modules/terraform/terraform-backend.variant

```diff
@@ -52,27 +52,25 @@ job "terraform backend generate component" {
     }
   }
 
-  exec {
-    command = "jq"
-    args = [
-      "-M",
-      "-n",
-      jsonencode(
-        {
-          "terraform" = {
-            "backend" = conf.stack-config
-          }
+  variable "data" {
+    type  = string
+    value = jsonencode(
+      {
+        "terraform" = {
+          "backend" = conf.stack-config
         }
-      )
-    ]
+      }
+    )
   }
 
-  log {
-    collect {
-      condition = event.type == "exec"
-      format    = event.exec.args[0]
-    }
```

Comment:
Why was this not writing to file?

Reply:
(e.g. it was working? no - before we moved to `jq`)

Reply:
it was writting `-M`, the first argument